R/example.R

```diff
@@ -0,0 +1,43 @@
+pieEx1 = list(
```

Comment:
Sorry, it's my fault. It was just for testing on my own. I will give a simple example.